dev-python/flask-paginate/flask-paginate-0.6.0.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( pypy3 python3_{6,7} )
+
+inherit distutils-r1
+
+MY_COMMIT="510ad833106134711868653fb597bf75ea8ac34f"
+
+DESCRIPTION="Pagination support for flask"
+HOMEPAGE="https://flask-paginate.readthedocs.io"
+# https://github.com/lixxu/flask-paginate/issues/68
+SRC_URI="https://github.com/lixxu/flask-paginate/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
```

Comment:
0.7.0 is the latest version, and this is not correct, you should use `SRC_URI="https://github.com/lixxu/flask-paginate/archive/${PV}.tar.gz"`

Reply:
Please remove that comment line also, now that the issue have been fixed it only disturbs :) 

Thank you for your effort.